cfgov/login/forms.py

```diff
@@ -1,75 +1,19 @@
-import time
 from datetime import timedelta
 
-from django.conf import settings
 from django.contrib.auth import get_user_model
-from django.contrib.auth.forms import SetPasswordForm
 from django.core.exceptions import ObjectDoesNotExist, ValidationError
 from django.utils import timezone
 
-from wagtail.admin.forms.auth import (
-    AuthenticationForm,
-    PasswordChangeForm,
-    PasswordResetForm,
-)
+from wagtail.admin.forms.auth import AuthenticationForm
```

Comment:
Should we inherit from Wagtail's [`LoginForm`](https://github.com/wagtail/wagtail/blob/70a6230a650f8ce299d6f883483c37bacfc325d6/wagtail/admin/forms/auth.py#L8) instead? It improves the styling somewhat:

|Current|Using `LoginForm`|
|-|-|
|<img width="1552" alt="image" src="https://user-images.githubusercontent.com/654645/231817688-4fa1ad61-524b-4b8d-abc4-273b7c997bb3.png">|<img width="1552" alt="image" src="https://user-images.githubusercontent.com/654645/231818934-66669fce-1c18-4345-b6b4-0328a3961477.png">|

`LoginForm` also brings in the field for the "Remember me" checkbox, so I'm guessing that won't work unless we make this change.

There's another issue which is that the "Forgotten password?" text is supposed to be in the password field, not below it:

<img width="1432" alt="image" src="https://user-images.githubusercontent.com/654645/231819691-5c48a817-db13-4ebc-b132-5734ee9373c3.png">

We could delete [these lines](https://github.com/cfpb/consumerfinance.gov/blob/7b6190d4c43c8fcffb5a126e1285ecb1e731344d/cfgov/wagtailadmin_overrides/static/css/global.css#L49-L52) as part of this PR to fix this as well.